File: MedicApp/Models/DbPersonInfo.cs
namespace MedicApp.Models
{
    public class DbPersonInfo
    {
        public Guid Id { get; set; }
        public string? FirstName { get; set; }
        public string? LastName { get; set; }
        public DateTime Birthday { get; set; }
        public string JMBG { get; set; }
        public string Address { get; set; }




        public bool IsDeleted { get; set; }
        public DateTime CreatedAt { get; set; }

            

    }
}

File: Client/src/app/model/DbPersonInfo.ts
export interface DbPersonInfoInterface {
  id: number;
  firstName: string;
  lastName: string;
  birthday: Date;
  jmbg: string;
  address: string;
  isDeleted: boolean;
  createdAt: Date;
}

export class DbPersonInfo implements DbPersonInfoInterface {
  id: number;
  firstName: string;
  lastName: string;
  birthday: Date;
  jmbg: string;
  address: string;
  isDeleted: boolean;
  createdAt: Date;

  constructor(obj: DbPersonInfoInterface) {
    this.id = obj.id;
    this.firstName = obj.firstName;
    this.lastName = obj.lastName;
    this.birthday = obj.birthday;
    this.jmbg = obj.jmbg;
    this.address = obj.address;
    this.isDeleted = obj.isDeleted;
    this.createdAt = obj.createdAt;
  }
}
